src/megaclient.cpp

```diff
@@ -6152,6 +6157,43 @@ void MegaClient::sc_la()
     }
 }
 
+void MegaClient::sc_ub()
+{
+    bool statusRecv = false;
+    int status = -2;
+    for (;;)
+    {
+        switch (jsonsc.getnameid())
+        {
+            case 's':
+                statusRecv = true;
+                status = (int)jsonsc.getint();
+                break;
+
+            case EOO:
+                if (!statusRecv)
+                {
+                    std::string err = "Missing status in `ub` action packet";
+                    LOG_warn << err;
+                    sendevent(99449, err.c_str(), 0);
+                    return;
+                }
+
+                businessStatus = status;
```

Comment:
What about the other state's flags? Please, make them consistent.
```suggestion
                businessStatus = status;
                business = bool(status);
                if (!status)
                {
                    businessMaster = false;
                }
```